src/app/dataset-view/additional-components/metadata-component/components/form-components/order-field/order-field.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+
+import { OrderFieldComponent } from "./order-field.component";
+import { TooltipIconComponent } from "src/app/dataset-block/metadata-block/components/tooltip-icon/tooltip-icon.component";
+import { NgbTooltipModule } from "@ng-bootstrap/ng-bootstrap";
+import { FormGroup, FormsModule, ReactiveFormsModule } from "@angular/forms";
+
+describe("OrderFieldComponent", () => {
+    let component: OrderFieldComponent;
+    let fixture: ComponentFixture<OrderFieldComponent>;
```

Comment:
No tests for data rendering, selection influence on form values

Reply:
Added tests data rendering, selection influence on form values